tutorials/dapps/papi/remark-tutorial.md

```diff
@@ -0,0 +1,169 @@
+                console.log(chalk.black.bgRed(`You got mail!`));
+                console.log(chalk.black.bgCyan("From:"), chalk.bold.whiteBright(sender.toString()));
+                console.log(chalk.black.bgBlue("Hash:"), chalk.bold.whiteBright(hash.asHex()));
+            }
+        });
+    } else {
+        // If the account is not provided, we exit
+        console.error('Account is required');
+        return;
+    }
+```
+
+### Compiling and Running
+
+Compile and execute the application using the following command:
+
+```shell
+npm start -- --account <account-address>
+```
+
+For example:
+
+```shell
+❯ npm start -- --account 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+
+> youve-got-mail-web3@1.0.0 start
+> tsc && node ./dist/index.js --account 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+
+ __        __   _    _____   __  __       _ _  __        __    _       _
+ \ \      / /__| |__|___ /  |  \/  | __ _(_) | \ \      / /_ _| |_ ___| |__   ___ _ __
+  \ \ /\ / / _ \ '_ \ |_ \  | |\/| |/ _` | | |  \ \ /\ / / _` | __/ __| '_ \ / _ \ '__|
+   \ V  V /  __/ |_) |__) | | |  | | (_| | | |   \ V  V / (_| | || (__| | | |  __/ |
+    \_/\_/ \___|_.__/____/  |_|  |_|\__,_|_|_|    \_/\_/ \__,_|\__\___|_| |_|\___|_|
+
+Watching account: 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+[smoldot] Smoldot v2.0.34
+[smoldot] Chain initialization complete for westend2. Name: "Westend". Genesis hash: 0xe143…423e. Chain specification starting at: 0x10cf…b908 (#23920337)
+```
+
+### Testing the CLI
+
+To test the application, navigate to the [PAPI Dev Console > Extrinsics](https://dev.papi.how/extrinsics#networkId=westend&endpoint=light-client). Select the `System` pallet and the `remark_with_event` call.
+
+Ensure the input field follows the convention `address+email`. For example, if monitoring `5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY`, the input should be:
+
+![Screenshot 2025-03-03 at 4.58.04 PM](https://hackmd.io/_uploads/S1nx8omo1x.png)
+
+Submit the extrinsic and sign it using the Polkadot.js browser wallet. The CLI will display the following output and play the "You've Got Mail!" sound:
+
+```
+ __        __   _    _____   __  __       _ _  __        __    _       _
+ \ \      / /__| |__|___ /  |  \/  | __ _(_) | \ \      / /_ _| |_ ___| |__   ___ _ __
+  \ \ /\ / / _ \ '_ \ |_ \  | |\/| |/ _` | | |  \ \ /\ / / _` | __/ __| '_ \ / _ \ '__|
+   \ V  V /  __/ |_) |__) | | |  | | (_| | | |   \ V  V / (_| | || (__| | | |  __/ |
+    \_/\_/ \___|_.__/____/  |_|  |_|\__,_|_|_|    \_/\_/ \__,_|\__\___|_| |_|\___|_|
+
+Watching account: 5Cm8yiG45rqrpyV2zPLrbtr8efksrRuCXcqcB4xj8AejfcTB
+You've got mail!
+From: 5Cm8yiG45rqrpyV2zPLrbtr8efksrRuCXcqcB4xj8AejfcTB
+Hash: 0xb6999c9082f5b1dede08b387404c9eb4eb2deee4781415dfa7edf08b87472050
+```
+
+### Conclusion
+
+This application can be extended in various ways, such as implementing a chatroom using remarks or leveraging rollups on Polkadot to enhance functionality.
```

Comment:
Using remarks for NFTs, inscriptions, chatroom features is frowned upon as they are not an efficient design for these applications. I would not put such recommendations on the official docs. 

Would rephrase it to something like this:
"This application shows how Polkadot API can be used in building applications. Explore all of the functions exposed by PAPI here: "

Reply:
Addressed